src/commands/lib/get-package-jsons.js

```diff
@@ -0,0 +1,17 @@
+const FileHound = require('filehound');
+const jsonfile = require('jsonfile');
+const pMap = require('p-map');
+
+module.exports = ({ folder, concurrency }) =>
+    FileHound.create()
```

Comment:
never knew about this package but loveee this chained syntax

Reply:
yeah, it's freak'n awesome. defo need to migrate other tools to use this!